buildutils/src/utils.ts

```diff
@@ -75,5 +75,5 @@ export function run(
   if (value === null) {
     return '';
   }
-  return value.toString().replace(/\n$/, '');
+  return value.toString().trim();
```

Comment:
I used `trimRight()` to ensure full compat, but unless there is any significant leading whitespace this should be good as well!

Reply:
Oh ha, I see, `trimRight` is an alias

Reply:
It seems only `trimRight()` is supported on Node:
```bash
PS C:\dev> node
> ''.trimEnd()
TypeError: "".trimEnd is not a function
    at repl:1:4
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:240:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:441:10)
    at emitOne (events.js:121:20)
    at REPLServer.emit (events.js:211:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
> ''.trimRight()
''
>
```

Reply:
I just said heck with it and went with a more explicit regex.